data/2fa/domain.yml

```diff
@@ -0,0 +1,72 @@
+version: "3.1"
+
+intents:
+  - intent_enable_2fa
+  - intent_enable_2fa_personal
+  - intent_enable_2fa_org_admin_info
+  - intent_individual_2fa_form_redirect
+  - intent_enable_2fa_org_admin_setup
+  - intent_dispatch_action_enable_org_2fa
+
+entities: []
+
+slots: {}
+
+actions:
+  - action_enable_2fa
+  - action_enable_org_2fa
+
+responses:
+  utter_user_is_org_admin:
+    - text: "It looks like you're an Org Admin. Do you want to enable two-factor authenticaiton for yourself or your entire org?"
+      buttons:
+        - title: Enable two-factor authentication for my account only
+          payload: /intent_enable_2fa_personal
+        - title: Enable two-factor authentication for my org
+          payload: /intent_enable_2fa_org_admin_info
+  utter_enable_2fa_individual_1:
+    - text: "I can provide information on how to set up two factor authentication for your personal account."
+  utter_enable_2fa_individual_2:
+    - text: >
+        Here is how the process works:
+        1. I will open a link to sso.redhat.com for you.
+        2. In that page, there will be a section named *Two-factor authentication*.
+        3. In that section, there is a button named *Set up Authenticator application*.
+        4. Once you click that button, you will be asked to reauthenticate.
+        5. After you reauthenticate, you will be redirected to a form to set up two-factor authentication.
+  utter_enable_2fa_individual_3:
+    - text: "Do you wish to continue?"
+      buttons:
+        - title: Yes
+          payload: /intent_individual_2fa_form_redirect
+        - title: No
+          payload: /intent_closing_start
```

Comment:
Users will have the option to use yes /  no  (in writing) - we should use intent_core_yes and intent_core_no here as well and create rules/stories to continue with the flow